contrib/build_sysimg.jl

```diff
@@ -12,6 +12,18 @@ function default_sysimg_path(debug=false)
     end
 end
 
+"""
+    build_sysimg(sysimg_path=default_sysimg_path, cpu_target="native", userimg_path=nothing; force=false)
```

Comment:
nice. the warning about this has been bugging me for a while, glad you found a way to fix it